packages/ui/src/views/repo/pull-request/details/components/conversation/sections/pull-request-comment-section.tsx

```diff
@@ -7,32 +7,33 @@ interface PullRequestMergeSectionProps {
   handleAction?: () => void
 }
 const PullRequestCommentSection = ({ commentsInfo, handleAction }: PullRequestMergeSectionProps) => {
+  const isSuccess = commentsInfo.status === 'success'
+
   return (
     <Accordion.Item value="item-2">
-      <Accordion.Trigger className="text-left" hideChevron>
+      <Accordion.Trigger
+        className="py-3 text-left [&>svg]:-rotate-0 [&>svg]:data-[state=open]:-rotate-180"
+        chevronClassName="text-icons-3 self-start mt-1"
+        hideChevron
+      >
         <StackedList.Field
+          className="flex gap-y-1"
           title={
             <LineTitle
+              textClassName={isSuccess ? '' : 'text-foreground-danger'}
               text={commentsInfo.header}
               icon={
-                commentsInfo.status === 'success' ? (
-                  <Icon name="success" className="text-foreground-success" size={16} />
-                ) : (
-                  <Icon name="triangle-warning" className="text-destructive" />
-                )
+                <Icon
+                  className={isSuccess ? 'text-icons-success' : 'text-icons-danger'}
+                  name={isSuccess ? 'success' : 'triangle-warning'}
+                />
               }
             />
           }
-          description={commentsInfo.content && <LineDescription text={commentsInfo.content} />}
+          description={!!commentsInfo?.content && <LineDescription text={commentsInfo.content} />}
         />
-        {commentsInfo.status === 'failed' && (
-          <Text
-            onClick={() => {
-              handleAction?.()
-            }}
-            className="pr-2"
-            size={1}
-          >
+        {commentsInfo.status === 'failed' && !!handleAction && (
+          <Text className="pr-2" onClick={() => handleAction?.()} size={1}>
```

Comment:
Probably can drop the optional chaining here since its existence was proven on the line before
```suggestion
          <Text className="pr-2" onClick={() => handleAction()} size={1}>
```